packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -111,7 +106,7 @@ class Auth {
     );
   }
 
-  Future<void> login(void Function(String) prompt) async {
+  Future<void> getCredentials(void Function(String) prompt) async {
```

Comment:
Should this be called `loadCredentials`? It's strange that it's not returning the credentials based on the naming.

Reply:
I still think it's strange for `auth` to not expose a `login` and `signup` since that's what most auth abstractions do (e.g. FirebaseAuth). Currently we also still have `logout` as a public method on `Auth` which makes things confusing imo. Is the main benefit of this change to avoid having to pass the two prompts to the signup call? I think the flow is a bit confusing in this state.

Reply:
Re: naming, totally agreed.  `loadCredentials` is a better name (or `fetchCredentials`), and `logout` should probably be `clearCredentials`.

Let me take a stab at the login/signup approach. I was avoiding this since the flow is somewhat interactive, but it might not be so bad.

Reply:
I think overall it'll be easier to reason about because I feel Auth shouldn't expose low-level implementation details about loading/clearing credentials and should instead expose higher abstractions like login, logout, signup, etc.

Reply:
Updated